src/utils.ts

```diff
@@ -32,3 +32,11 @@ export function countWords(text: string) {
 }
 
 export const numberfyId = (id: ID) => typeof id === 'string' ? parseInt(id, 10) : id;
+
+export const assertWindow = (message = 'BUG: Window is undefined') => {
```

Comment:
i think you can just assume we're in a browser in this lib, at least for now

Reply:
Yeah I was thinking of possible future REX use but I'll yank it for now